renovate.json

```diff
@@ -37,6 +37,13 @@
        "matchManagers": [ "pip_requirements" ],
        "groupName": "pip_requirements"
     },
+    {
+      "description": "Group go updates",
+      "matchManagers": ["gomod"],
+      "matchDepTypes": ["major", "minor", "patch", "digest", "indirect"],
+      "groupName": "go updates",
+      "postUpdateOptions": ["gomodUpdateImportPaths", "gomodTidy"]
+    },
```

Comment:
What does this change do? This is not part of the PR description.

Reply:
Sorry. It shouldn't be here. Removing